app/controllers/admin/consultation_responses_controller.rb

```diff
@@ -0,0 +1,53 @@
+class Admin::ConsultationResponsesController < ApplicationController
+	layout "admin_panel_sidenav"
+  before_action :authenticate_user!
+	before_action :require_admin, only: [:index, :show, :approve, :reject]
+	before_action :set_consultation_response, only: [:show, :approve, :reject]
+
+	def index
+	@consultation_responses = ConsultationResponse.all.order(created_at: :desc).published_consultation.filter_by(params[:page], filter_params.to_h, sort_params.to_h)
```

Comment:
```suggestion
	@consultation_responses = ConsultationResponse.all.order(created_at: :desc).published_consultation.filter_by(params[:page], filter_params.to_h, sort_params.to_h)
```

Use the `sort_params` instead of `order`

Reply:
I am not sure how this can be accomplished as this seems custom built functionality of Commutatus. Please take care of it at your end.

Reply:
@kanth5 - here is the documentation - https://guides.rubyonrails.org/active_record_validations.html#custom-methods

Reply:
Mike, we are not sure of how to use 'sort_params' to pass 'created_at' in 'desc' order and have not made any changes to the above code other than removing 'all' as suggested. Please have it taken care of at your end.